frontend/src/router.ts

```diff
@@ -305,4 +305,29 @@ export const routerBeforeEach: NavigationGuard = async (to, from, next) => {
   next();
 };
 
-router.beforeEach(routerBeforeEach);
+export const redirectSections: NavigationGuard = async (to, from, next) => {
+  if (_.get(to, "meta.activator", false)) {
+    const activatorChildren = getRootRouteChildren().find(
+      (route: RouteConfig) => `/${route.path}` === `${to.fullPath}/`
+    ).children;
+    const userClaims = await getUserClaims();
```

Comment:
By the way, why are we using local function if we have `getUserClaims` action in the `user` store module?